src/Avalonia.Controls/ApplicationLifetimes/ActivationKind.cs

```diff
@@ -0,0 +1,8 @@
+namespace Avalonia.Controls.ApplicationLifetimes;
+
+public enum ActivationKind
+{
+    OpenUri = 20,
+    Reopen = 30, // i.e. Dock Icon clicked on MacOS
+    Background = 40 // i.e. Entered or left background.
```

Comment:
nit: doc comments here and on other new public types

Reply:
commented public api.